### PR TITLE
MLR-251: update get monitoring location by agency and site number to

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
return single monitoring location object instead of an array of one
monitoring location.  Fix associated tests.

### CODE REVIEW

File: src/test/java/gov/usgs/wma/mlrlegacy/ControllerTest.java [hunk 1]
```diff
@@ -57,15 +57,13 @@
 	private SecurityContext securityContext;
 
 	@Test
-	public void givenReturnData_whenGetByMap_thenReturnList() throws Exception {
-		List<MonitoringLocation> mlList = new ArrayList<>();
+	public void givenReturnData_whenGetByAK_thenReturnMonitoringLocation() throws Exception {
 		MonitoringLocation mlOne = new MonitoringLocation();
 
 		mlOne.setId(BigInteger.ONE);
```
Comment (dsteinich) [inline src/test/java/gov/usgs/wma/mlrlegacy/ControllerTest.java]: Are line 63-69 left over artifacts from a couple versions ago?

Comment (kmschoep-usgs) [reply src/test/java/gov/usgs/wma/mlrlegacy/ControllerTest.java]: I don't know?  maybe?

**Comment by dsteinich** [comment]
Almost forgot - the application.yml will need to following (or revert to profile to default on the swagger config):

spring:
  profiles:
    active: default, swagger

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -14,7 +14,9 @@ spring:
     username: ${mlrLegacyDataUsername}
     password: ${mlrLegacyDataPassword}
     driver-class-name: org.postgresql.Driver
-
+  profiles:
```
Comment (kmschoep-usgs) [inline src/main/resources/application.yml]: @dsteinich like this?

Comment (dsteinich) [reply src/main/resources/application.yml]: Yes

**Pull request merged**